src/bug_report.rs

```diff
@@ -0,0 +1,198 @@
+use crate::utils::exec_cmd;
+use std::fs;
+use std::path::PathBuf;
+
+pub fn create() {
+    let os_info = os_info::get();
+
+    let environment = Environment {
+        os_type: os_info.os_type(),
+        os_version: os_info.version().to_owned(),
+        shell_info: get_shell_info(),
+        starship_config: get_starship_config(),
+    };
+
+    let link = make_github_issue_link(crate_version!(), environment);
+
+    if open::that(&link)
+        .map(|exit_status| exit_status.success())
+        .unwrap_or(false)
+    {
+        println!(
+            "I was unable to launch your browser. You'll have to copy this link instead:\n\n{}",
+            link
```

Comment:
The link ends up being pretty huge, filling up my entire terminal emulator. I think this would be a good time to use a [hyperlink](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda) (or maybe generating a https://git.io/ shortened link 🤔 )

Reply:
I went with using git.io. It feels like a better solution, especially considering not all terminal emulators supporting hyperlinks (though I must admit support is impressive).